aid/pages/_template.jsx

```diff
@@ -9,6 +9,7 @@ import { rhythm } from '../utils/typography'
 
 import '~/styles/index'
 import Navbar from '~/components/Navbar'
+import Footer from 'components/Footer'
```

Comment:
Let's make it consistent by either making it `~/components/Footer` or removing it from the above imports.  
Thoughts?